### PR TITLE
DESKTOPAPP-1615: Bump sync-service to 3.9.0

### PR DESCRIPTION
**Pull request by SNegi01**
Changed version of sync-service to 3.9.0 for Oxford release.

### CODE REVIEW

File: docker-compose/7.1.N-docker-compose.yml
Comment (abhishekkumar31) [inline docker-compose/7.1.N-docker-compose.yml:167]: Replace ths SS version with 3.9.0 instead of 3.9.2

Comment (SNegi01) [reply docker-compose/7.1.N-docker-compose.yml]: version changed to 3.9.0.

**Review by abhishekkumar31 — APPROVED** [review]
LGTM

**Comment by alxgomz** [comment]
Guys, this PR actually contains errors or is not following our std way of working.
Can you please make sure to get approval from one of the devops team members (@gionn , @slohe1 or myself) before merging.

(e.g. here versions compat do not match the matrix and we have changed the sync service chart and some values are using the old names and more updates are needed there: https://github.com/Alfresco/alfresco-helm-charts/tree/main/charts/alfresco-sync-service)

**Pull request merged**
